TrenDAP/wwwroot/TypeScript/Features/DataSets/DataSets.tsx

```diff
@@ -25,127 +25,174 @@
 import * as React from 'react';
 import { TrenDAP, Redux } from '../../global';
 import { useAppSelector, useAppDispatch } from '../../hooks';
-import Table from '@gpa-gemstone/react-table/lib/index'
+import { ReactTable } from '@gpa-gemstone/react-table'
 import { Sort, FetchDataSets, SelectDataSetsStatus, RemoveDataSet, SelectDataSetsForUser, SelectDataSetsAllPublicNotUser, SelectDataSetsSortField, SelectDataSetsAscending, CloneDataSet, New } from './DataSetsSlice';
 import moment from 'moment';
-import DataSetData from './DataSetData';
 import { DNA, TrashCan, HeavyCheckMark, Pencil, Wrench } from '@gpa-gemstone/gpa-symbols';
-import { Warning } from '@gpa-gemstone/react-interactive'
+import { Warning, ToolTip } from '@gpa-gemstone/react-interactive'
+import { useNavigate } from "react-router-dom";
+
+
+type Hover = ('Pencil' | 'Clone' | 'Delete' | 'None')
 
 const DataSets: React.FunctionComponent = (props: {}) => {
     const dispatch = useAppDispatch();
+    const navigate = useNavigate();
     const DataSets = useAppSelector((state: Redux.StoreState) => SelectDataSetsForUser(state, userName));
     const publicDataSets = useAppSelector((state: Redux.StoreState) => SelectDataSetsAllPublicNotUser(state, userName));
 
     const [deleteItem, setDeleteItem] = React.useState<TrenDAP.iDataSet>(null);
+    const [hover, setHover] = React.useState<Hover>('None')
+    const cardHeaderRef = React.useRef<HTMLDivElement>(null);
+    const [cardHeaderHeight, setCardHeaderHeight] = React.useState<number>(50)
```

Comment:
Do we really need to do this via measurment?

Reply:
I did that before due to the card-headers height in each table not matching. However, after removing the cardHeaderHeight state it seems like this is not an issue anymore.